app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= devise_error_messages! %>
+
+
+  <div class="field">
+    <%= f.label :username %><br />
+    <%= f.text_field :username, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true %>
```

Comment:
It doesn't make much sense to have two fields using autofocus, remove one of the two, as I am unsure which one will actually receive focus on load.